matchbook-sdk-core/src/main/java/com/matchbook/sdk/core/model/dataobjects/LoginEnvelope.java

```diff
@@ -0,0 +1,29 @@
+package com.matchbook.sdk.core.model.dataobjects;
+
+import com.matchbook.sdk.core.model.dataobjects.auth.User;
+
+public class LoginEnvelope extends Envelope {
+
+    private User user;
+
+    public User getUser() {
+        return user;
+    }
+
+    private LoginEnvelope(Builder builder) {
```

Comment:
Constructors should be inserted before other methods, even if private, unless we changed our code conventions.